src/web/modules/payment_links/payment_links.http.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response, NextFunction } from 'express'
+import { Products, AdminUsers } from '../../../lib/pay-request'
+
+function indexPaymentLinksByType(paymentLink: any): any {
+  const index = paymentLink._links.reduce((aggregate: any, linkDetails: any) => {
+    aggregate[linkDetails.rel] = linkDetails.href
+    return aggregate
+  }, {})
+  paymentLink._indexedLinks = index
+  return paymentLink
+}
+
+export async function list(req: Request, res: Response, next: NextFunction): Promise<void> {
+  try {
+    let account
+    const { accountId } = req.params
+
+    const paymentLinksRepsonse = await Products.paymentLinksByGatewayAccount(accountId)
+    const paymentLinks = paymentLinksRepsonse
+      .filter((link: any) => !(['PROTOTYPE', 'DEMO'].includes(link.type)))
```

Comment:
I'd potentially reverse this to be `link.type === 'ADHOC'` as I think it's a bit clearer, but it's fine if you think otherwise.